animated-base/src/main/java/com/facebook/imagepipeline/animated/base/AbstractAnimatedDrawable.java

```diff
@@ -96,6 +96,9 @@
   private boolean mInvalidateTaskScheduled;
   private long mNextFrameTaskMs = -1;
 
+  private boolean resumable = false;
+  private int mPauseFrameNumber = 0;
```

Comment:
Is it not possible to keep `mScheduledFrameNumber`? It shouldn't be modified when the animation is paused.